CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ include_directories(${CMAKE_BINARY_DIR})
 if(NOT ITK_SOURCE_DIR)
   include(ITKModuleExternal)
 else()
+  set(ITK_DIR ${CMAKE_BINARY_DIR})
```

Comment:
@thewtex Why is this required? Without this some random ITK installation is usually found.

Reply:
If built from `ITK/Modules/Remote`, we want `find_package(ITK` in `PerformanceBenchmarking/examples` to find the ITK we are building.

Reply:
Should that be in the [template](https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/CMakeLists.txt) then?

Reply:
Yes.